.scripts/linux/build.sh

```diff
@@ -10,6 +10,6 @@ cd $REPO_DIR
 # Build reinforcement_learning
 mkdir -p build
 cd build
-cmake ..
+cmake .. -Drlclientlib_BUILD_ONNXRUNTIME_EXTENSION=On
```

Comment:
should we have two separate ci's building with and without?

Reply:
Good idea, but would prefer to avoid expanding the scope of this PR (and there may a bit of a chicken-and-egg problem with setting it up so that it works as part of a PR that includes this change)